### ISSUE TITLE
Abnormal circulating valproic acid level

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5985]
**Preferred term label:** Abnormal circulating valproic acid level

**Synonyms** Abnormal VPA level

**Definition (free text, please give PubMed ID)** Concentration of valproic acid in the blood is outside the normal range., PMID: 6416845; PMID: 17448293

**Parent term (use hpo.jax.org/app)** Abnormal circulating monocarboxylic acid cocentration HP:0010996

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating valproic acid level

**Synonyms** Decreased VPA level

**Definition (free text, please give PubMed ID)** Concentration of valproic acid in the blood is above the normal range., PMID: 6416845

**Parent term (use hpo.jax.org/app)**  Abnormal circulating valproic acid level

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:**  Increased circulating valproic acid level

**Synonyms** Increased VPA level

**Definition (free text, please give PubMed ID)** Concentration of valproic acid in the blood is above the normal range., PMID: 6416845

**Parent term (use hpo.jax.org/app)**  Abnormal circulating valproic acid level

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Valproic acid is an antiepileptic drug, and so there is no "abnormal" level, it is a matter of dosing. This is not in the scope of HPO.
